scripts/tests/calibnet_offline_rpc_check.sh

```diff
@@ -43,5 +43,15 @@ if [ $exit_code -ne 0 ]; then
     exit 1
 fi
 
+# Compare the ws endpoints
+$FOREST_TOOL_PATH api compare "$snapshot" --forest /ip4/127.0.0.1/tcp/8080/http --lotus /ip4/127.0.0.1/tcp/8081/http --n-tipsets 5 --ws
```

Comment:
This works?
If we use the `--ws` flag we should pass `/ip4/127.0.0.1/tcp/8080/ws`

Reply:
Interesting.

Reply:
I'll investigate the issue separately, fixed the PR

Reply:
Sure.

Reply:
Our error handling is misguiding us here; we won't exercise web socket communication mode if we don't use `/ws` in urls when using the `--ws` flag.

I think we should validate things upfront.

Reply:
> I think we should validate things upfront.

@elmattic That makes sense, could you open an issue for that?

Reply:
I have a fix here: https://github.com/ChainSafe/forest/pull/4018